db/CMakeLists.txt

```diff
@@ -38,7 +38,7 @@ target_sources(db
     snapshot/backup_task.cc
     rate_limiter.cc
     per_partition_rate_limit_options.cc
-    row_cache.cc,
+    row_cache.cc
```

Comment:
Temporary change just to make things work, once [this PR](https://github.com/scylladb/scylladb/pull/22755) makes it into master it will be removed